src/components/Navbar/SettingsDrawer/TafsirSection.tsx

```diff
@@ -36,9 +36,7 @@ const TafsirSection = () => {
     () => (
       <div>
         {selectedTafsirs.map((id) => (
-          <Skeleton key={id}>
```

Comment:
and this tafsir component doesn't seems to be used in the settings drawer anymore. but it's okay :), lets get the changes in

Reply:
Yes, I had to add it to settings in order to test it as well 🙂 .  I added it just in case it's an upcoming feature.